src/vaadin-grid-drag-and-drop-mixin.html

```diff
@@ -148,11 +148,11 @@
           e.dataTransfer.setData('text', this.__formatDefaultTransferData(rows));
 
           row.setAttribute('dragstart', '');
+          this.updateStyles({
+            '--_grid-drag-start-x': `${e.clientX - rowRect.left + 20}px`,
+            '--_grid-drag-start-y': `${e.clientY - rowRect.top + 10}px`
+          });
           if (rows.length > 1) {
-            this.updateStyles({
-              '--_grid-drag-start-x': `${e.clientX - rowRect.left + 20}px`,
-              '--_grid-drag-start-y': `${e.clientY - rowRect.top + 10}px`
-            });
             row.setAttribute('dragstart', rows.length);
```

Comment:
Nit: can we combine two lines into one: 
`row.setAttribute('dragstart', rows.length > 1 ? rows.length : '');` and do it before the `updateStyles`?

Reply:
Done.